docs/admin/logical-replication.rst

```diff
@@ -87,16 +87,14 @@ clusters by defining its own publications.
 A cluster can have multiple subscriptions. It is also possible for a cluster to
 have both subscriptions and publications. A cluster cannot subscribe to locally
 already existing tables, therefore it is not possible to setup a bi-directional
-replication (both sides subscribing to ``ALL TABLES`` leads to a cluster trying to
-replicate its own tables from another cluster). However, two clusters still can
-cross-subscribe to each other if one cluster subscribes to locally non-existing
-tables of another cluster and vice versa.
+replication (both sides subscribing to ``ALL TABLES`` leads to a cluster trying
+to replicate its own tables from another cluster). However, two clusters still
+can cross-subscribe to each other if one cluster subscribes to locally
+non-existing tables of another cluster and vice versa.
 
 A subscription is added using the
 :ref:`CREATE SUBSCRIPTION <sql-create-subscription>` command and can be
-stopped/resumed at any time using the
-:ref:`ALTER SUBSCRIPTION <sql-alter-subscription>` command and removed using the
-:ref:`DROP SUBSCRIPTION <sql-drop-subscription>` command.
+removed using the :ref:`DROP SUBSCRIPTION <sql-drop-subscription>` command.
```

Comment:
Maybe lets add:
A subscription starts replicating on its creation and stops on its removal (if no failure happen in-between).